src/docs-assets/styles.scss

```diff
@@ -111,37 +99,34 @@ header {
     background: $mw-bg-gradient-2-horizontal;
   }
   &.text-primary {
-    background: $mw-color-text-primary;
+    background: $mw-neutral-text;
   }
   &.text-secondary {
-    background: $mw-color-text-secondary;
+    background: $mw-neutral-gfx;
   }
   &.bg-primary {
-    background: $mw-color-background-primary;
+    background: $mw-background-primary;
```

Comment:
We have:

```
$mw-background-primary: $MW_COLOR_GRAY_6;
$mw-background-secondary: $MW_COLOR_GRAY_5;
$mw-background-inverted: $MW_COLOR_GRAY_2;
```

Are you suggesting `mw-background-plain` instead of `$mw-background-primary`? If that's what you mean, how should the secondary be named?